src/java/jni.rs

```diff
@@ -997,4 +999,132 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn check_gen_jni_function_namespace_and_cfg() {
+        let func_name = "testDummyFuncName";
+        let native_name = "test_dummy_func_name";
+        let mut ctx = Context::default();
+        let mut dummy_outputs: HashMap<String, String> = HashMap::new();
```

Comment:
Type inference can deduce this type, so I'd suggest replacing this with
```suggestion
        let mut dummy_outputs = HashMap::new();
```